VTOrc converts a tablet to DRAINED type if it detects errant GTIDs on it (#13873)
vitessio/vitess: changelog/18.0/18.0.0/summary.md

should start using this flag. By default, VTOrc will be able to run `EmergencyReparentShard`. The users must specify the
flag to `false` to change the behaviour.

#### <a id="new-flag-errant-gtid-convert"/>VTOrc flag `--convert-tablets-with-errant-gtids`

> VTOrc has a new flag `--convert-tablets-with-errant-gtids` that allows the users to toggle the ability of VTOrc to change the 

Reviewer: ```suggestion
VTOrc has a new flag `--convert-tablets-with-errant-gtids` that allows users to choose whether VTOrc should change the 
```

Author: Fixed

tablet type of tablets with errant GTIDs to `DRAINED`. By default, the flag is false.

> This feature allows the users to configure VTOrc such that any tablet that encounters errant GTIDs is automatically taken out of the

Reviewer: ```suggestion
This feature allows users to configure VTOrc such that any tablet that encounters errant GTIDs is automatically taken out of the
```

Author: Fixed
